File: CustomerService/src/main/java/com/example/customerservice/Address/Address.java
package com.example.customerservice.Address;

import lombok.Data;

@Data
public class Address {
    private String _id;
    private String street;
    private String streetNumber;
    private Number zepCode;
    private String city;
    private String country;
    private String homePhone;
    private String type;
}

File: ShoppingCartService/Models/Address/Address.cs
namespace ShoppingCartService.Models{
    public class Address{
        public string? id{get; set;}
        private string? street{get; set;}
        private string? streetNumber{get; set;}
        private long? zepCode{get; set;}
        private string? city{get; set;}
        private string? country{get; set;}
        private string? homePhone{get; set;}
        private string? type{get; set;}
    }
}